src/main/webapp/common.js

```diff
@@ -305,3 +305,20 @@ async function addMenu() {
     }
   });
 }
+
+/**
+ * Make a request to Google Books API to obtain the JSON with details about the
+ * book with given ID to be able to print the title and author of the book on
+ * the shopping cart page.
+ * @param {String} id 
+ */
+
+async function getBookJSON(id) {
+  // To get results from UK publishing houses.
+  const COUNTRY = 'UK';
+  let URL = `https://www.googleapis.com/books/v1/volumes/${id}`;
+  URL += `?country=${COUNTRY}`;
```

Comment:
What's with all the variables and concatenation? Why not have a single string?

Reply:
Done